linkerd/app/inbound/src/lib.rs

```diff
@@ -227,28 +232,33 @@ pub fn spawn<P>(
             DispatchDeadline::after(dispatch_timeout)
         }))
         .push(errors::layer())
-        .push(trace::layer(
-            |src: &Source| info_span!("source", peer_ip = %src.remote.ip(), peer_id=?src.tls_peer),
-        ))
+        .push(trace::layer(|src: &tls::accept::Meta| {
+            info_span!(
+                "source",
+                peer.ip = %src.addrs.peer().ip(),
+                peer.id = ?src.peer_identity,
```

Comment:
Nit/TIOLI: to me, `peer.id` _kind of_ implies an internal numeric identifier, while `peer.identity` suggests a cryptographic identity? but on the other hand, it's more characters in the logs, so maybe this is better...